tests/main_test.py

```diff
@@ -2,8 +2,10 @@
 import json
 import logging
 import pytest
+import time
```

Comment:
use pep8 coding style (python libs first). You can use your [editor](https://dev.to/adamlombard/how-to-use-the-black-python-code-formatter-in-vscode-3lo0) to do this automatically